css/selectors/-moz-last-node.json

```
@@ -12,7 +12,7 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": "≤72"
+              "version_added": "≤69"
```

See: https://bugzilla.mozilla.org/show_bug.cgi?id=1337068#c27

```suggestion
              "version_added": "55"
```